---
layout: page
title: "Algae, bacteria, and the genetics in between"
description: Bioinformatic research on bacterial genome architecture with regards to microbial interactions
img: assets/img/projects/t4ss/ehuxPinhibens.jpg
importance: 1
category: Science
giscus_comments: true
toc:
  sidebar: left

---

This project is an M.Sc research project that was part of my masters' thesis, I was under the supervision of Dr. Einat Segev and have also been mentored by PhD student Yemima Duchin Rapp, in the Plant & Environmental Sciences department of the Weizmann Institute of Science. 

Parts of my work were featured [in a very recent paper which was submitted not long ago](https://www.biorxiv.org/content/10.1101/2023.11.14.567099v2). Therefore I'll only give very brief and partial introduction before diving into the data analysis driven portion of the project. Please read the paper if you want to read about the biology! :man_scientist: :microbe:

Not a biologist? No worries! You can [skip the biology and go straight into my data analysis project!](#Defining a research question and approach).

<div class="row justify-content-center">
    <div class="col-sm mt-3 mt-md-0 text-center">
        {% include figure.html path="assets/img/projects/t4ss/barentsbloom.jpg" title="Phytoplankton bloom, Barents Sea." class="img-fluid rounded z-depth-1" %}
    </div>
</div>
<div class="caption">
    Phytoplankton bloom, Barents Sea. August 3rd, 2023. These seasonal blooms are where many different organisms interact, as well as the bacteria and micro-algae we study. The image above was acquired by the Moderate Resolution Imaging Spectroradiometer (MODIS) on NASA's Terra satellite.
</div> 

### Biological Introduction

Horizontal gene transfer (HGT), vital in bacterial evolution, involves the transfer of genetic material between unrelated bacteria. Despite small genomes, bacteria display remarkable genetic variations, enabling adaptability and flexibility. HGT, through mechanisms like Transformation, Transduction, Gene Transfer Agents, and Conjugation, empowers bacteria to acquire new capabilities for symbiosis, antibiotic resistance, metabolic activity, and pathogenicity.

A central mechanism for HGT is the type IV secretion system (T4SS), facilitating cell-to-cell contact and genetic material transfer. Many bacteria possess T4SS on plasmids or chromosomes, impacting their functionality. In the environmentally crucial Roseobacter group, T4SS genes vary in genomic locations, raising questions about their functional impact.

Conjugation, a form of HGT, is suggested to contribute significantly to the ecological success of Roseobacters, known for diverse metabolic capabilities and symbiotic interactions, especially with algal hosts. Algal exudates, influencing bacterial physiology, may act as signals promoting bacterial HGT.

Many Roseobacters interact with algal hosts, benefiting from algae's nutrient-rich exudates in the nutrient-poor ocean. This proximity supports nutrient flux for planktonic bacteria. Algal exudates impact bacterial physiology and interactions, promoting horizontal gene transfer (HGT) through bacterial proximity and attachment. Our study explores Roseobacter conjugation in the ecological context of the algal host *Emiliania huxleyi*. We utilized Roseobacter species identified in algal samples, testing their ability to conduct conjugation and examining the influence of T4SS genomic location. Additionally, we investigated the impact of algal exudates on bacterial conjugation, focusing on T4SS-encoding gene expression and DNA transfer.

This study explored Roseobacter conjugation in the context of an algal host, using *Emiliania huxleyi*. While a chromosome-encoded T4SS did not lead to successful conjugation, a plasmid-encoded T4SS functioned. Surprisingly, expression of plasmid-encoded T4SS genes did not increase in response to algal exudates, yet conjugation efficiency and plasmid transfer improved. This suggests a connection between bacterial response to an algal host and an increased likelihood of successful conjugation. The difference in functionality of a plasmid-encoded, or a chromosomally encoded T4SS sparked my interest and led to the project I'm going to show you now!'

<div class="row">
    <div class="col-sm mt-3 mt-md-0">
        {% include figure.html path="assets/img/projects/t4ss/ehux.jpg" title="Emiliania huxleyi" class="img-fluid rounded z-depth-1" %}
    </div>
    <div class="col-sm mt-3 mt-md-0">
        {% include figure.html path="assets/img/projects/t4ss/PIrosette.png" title="Phaeobacter inhibens bacteria form beautiful rosettes" class="img-fluid rounded z-depth-1" %}
    </div>
    <div class="col-sm mt-3 mt-md-0">
        {% include figure.html path="assets/img/projects/t4ss/ehuxPinhibens.jpg" title="Phaeobacter inhibens attached to an Emiliania huxleyi cell" class="img-fluid rounded z-depth-1" %}
    </div>
</div>
<div class="caption">
    On the left, Emiliania huxleyi, an abundant coccolithophore in modern oceans. Middle, multiple cells of Phaeobacter inhibens bacteria, forming what we call a 'rosette'. Right, Phaeobacter inhibens attached to an Emiliania huxleyi cell.
</div>

### Defining a research question and approach

### Materials and Methods


### Results
#### Sea Level Graphs from the Israeli National Mapping Agency Data

#### Relevant Tables and Graphs from the 2014 IPCC Report

#### QGIS Maps

### Discussion & Conclusions

### My thoughts, two years later


### References

